add-wildcard-domain

```diff
@@ -2587,6 +2587,7 @@ shiptrace-help.top
 shop-dofus.com
 simit-pagos.co
 simplylovingproducts.com
+site-7a0d7a3fa410.mypreview.site
```

Comment:
![image](https://github.com/user-attachments/assets/b3285c37-77c2-421a-8170-f9415c14cef4)

Reply:
site appears to 404 for me as well